test/message-generator/test/interop-proxy-with-multi-ups/interop-proxy-with-multi-ups.sh

```diff
@@ -0,0 +1,11 @@
+cd roles
+cargo llvm-cov --no-report -p pool_sv2
+cargo llvm-cov --no-report -p mining_proxy_sv2
+cargo build -p mining-device
+
+cd ../utils/message-generator/
+cargo build
```

Comment:
[ditto](https://github.com/stratum-mining/stratum/pull/905/files#r1603974488)

I could do this for every file, but it would be too repetitive.
I think you get the point